packages/react-core/src/demos/examples/DashboardWrapper.js

```diff
@@ -60,7 +60,8 @@ export default class DashboardWrapper extends React.Component {
       hasNoBreadcrumb,
       notificationDrawer,
       isNotificationDrawerExpanded,
-      hasPageTemplateTitle
+      hasPageTemplateTitle,
+      ...pageProps
```

Comment:
Is this addition to support something in the PR ?

Reply:
Not really. It was necessary in a previous version where I passed props like `isTertiaryNavGrouped`, `breadcrumbProps` to the `DashboardWrapper`.  But  as long as this PR is related to Page, I thought that we could keep this as it makes the wrapper more flexible.

Reply:
Agreed. It provides more flexibility.